Demo/Demo/FeatureViewControllers/CardDemoViewController.swift

```diff
@@ -89,6 +106,67 @@ class CardDemoViewController: FeatureBaseViewController, UITextFieldDelegate {
         ])
     }
 
+    // MARK: - FeatureBaseViewController Override
+
+    /// Allows us to only enable the pay button once all fields are filled out and we have an order ID
+    override func createOrderTapped(completion: @escaping (String?) -> Void = { _ in }) {
+        super.createOrderTapped { _ in
```

Comment:
we can get rid of `super` here - we want to call this class' implementation of `createOrderTapped`, even if it is inherited from a superclass

Reply:
oh it's the default parameter value. that syntax is kind of confusing.